### ISSUE TITLE
Not working with React Native 0.69

### ISSUE THREAD

**Issue by ammarahm-ed** [Grohden/react-native-cli-bump-version#17]
The command has stopped working after upgrading my project to RN 0.69. It seems that the property, `config.project.ios.pbxprojPath` has been removed from `react-native.config.js`. The following patch fixes this.

```diff
diff --git a/node_modules/react-native-cli-bump-version/react-native.config.js b/node_modules/react-native-cli-bump-version/react-native.config.js
index fe5274e..6197368 100644
--- a/node_modules/react-native-cli-bump-version/react-native.config.js
+++ b/node_modules/react-native-cli-bump-version/react-native.config.js
@@ -10,16 +10,14 @@ module.exports = {
                 console.log("My work here is done.");
                 return;
             }
-
             const appGradlePath = path.join(
                 config.project.android.sourceDir,
                 config.project.android.appName,
                 "build.gradle",
             );
-
             versioner({
                 root: config.root,
-                pbxprojPath: config.project.ios.pbxprojPath,
+                pbxprojPath: `${config.project.ios.sourceDir}/${config.project.ios.xcodeProject.name.replace(".xcworkspace",".xcodeproj")}/project.pbxproj`,
                 buildGradlePath: appGradlePath,
                 type: args.type,
                 semver: args.semver,
```

**Comment by Grohden** [comment]
@ammarahm-ed thx for reporting (and even the patch 🥇 ), I'm gonna take a look later today

**Comment by Grohden** [comment]
@ammarahm-ed, I've just released v1.5.0 with 3a5801ce17581bb3a56d9332cf0cc29fca42d36b, let me know if that works for you!

**Comment by ammarahm-ed** [comment]
It's working. Thanks 🚀
